model/keyvalue.go

```diff
@@ -25,8 +25,32 @@ const (
 	Float64Type = ValueType_FLOAT64
 	// BinaryType indicates the value is binary blob stored as a byte array
 	BinaryType = ValueType_BINARY
+
+	SpanKindKey     = "span.kind"
+	SamplerTypeKey  = "sampler.type"
+	SamplerParamKey = "sampler.param"
+)
+
+type SpanKind string
+
+const (
+	SpanKindClient      SpanKind = "client"
+	SpanKindServer      SpanKind = "server"
+	SpanKindProducer    SpanKind = "producer"
+	SpanKindConsumer    SpanKind = "consumer"
+	SpanKindInternal    SpanKind = "internal"
+	SpanKindUnspecified SpanKind = ""
```

Comment:
should this be unspecified - looks like some tests have expectations around that

Reply:
Unspecified is an OTEL value, it did not exist in OpenTracing, so I left it blank.

And in OTLP it's technically undefined which strings should be used because the values are numeric enums.